website/components/HelpDropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button, Dropdown, MenuProps, Modal } from "antd";
+import React, { ReactElement, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { DropdownSVG } from "../assets/icons";
+import { SecondaryButton } from "./Buttons";
+import { FlexColumnAlignCenter, FlexRowAlignCenter } from "./LandingPage/utils";
+
+// Defined in webpack config
+declare const WEBSITE3DCELLVIEWER_VERSION: string;
+declare const VOLUMEVIEWER_VERSION: string;
+
+export default function HelpDropdown(): ReactElement {
+  const [container, setContainer] = useState<HTMLDivElement | null>();
+  const containerRef = React.useRef<HTMLDivElement>(null);
+
+  const [showVersionModal, setShowVersionModal] = useState(false);
+
+  const items: MenuProps["items"] = [
+    {
+      key: "github",
+      label: (
+        <Link
+          to="https://github.com/allen-cell-animated/website-3d-cell-viewer"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          Visit GitHub repository
+        </Link>
+      ),
+    },
+    {
+      key: "github-issue",
+      label: (
+        <Link
+          to="https://github.com/allen-cell-animated/website-3d-cell-viewer/issues/new/choose"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          Report issue via GitHub
+        </Link>
+      ),
+    },
+    {
+      key: "forum",
+      label: (
+        <Link to="https://forum.allencell.org/c/software-code/11" target="_blank" rel="noreferrer noopener">
+          Allen Cell Discussion Forum
+        </Link>
+      ),
+    },
+    {
+      key: "version",
+      label: "Version info",
+      onClick: () => {
+        setShowVersionModal(true);
+      },
+    },
+  ];
+
+  // Use state update here to force a re-render so Dropdown is rendered with a valid `getContainer` callback.
+  // Otherwise, the `getContainer` callback will be undefined on the first render.
+  useEffect(() => {
+    setContainer(containerRef.current);
+  }, [containerRef.current]);
+  const getContainer = container !== null ? () => container! : undefined;
+
+  return (
+    <div ref={containerRef}>
+      <Dropdown menu={{ items: items }} getPopupContainer={getContainer} trigger={["click"]}>
+        <SecondaryButton>
+          <FlexRowAlignCenter $gap={6}>
+            Help <DropdownSVG />
+          </FlexRowAlignCenter>
+        </SecondaryButton>
+      </Dropdown>
+      <Modal
+        open={showVersionModal}
+        title="Version info"
+        getContainer={getContainer}
+        onCancel={() => {
+          setShowVersionModal(false);
+        }}
+        footer={() => {
+          return (
+            <Button
+              onClick={() => {
+                setShowVersionModal(false);
+              }}
+            >
+              Close
+            </Button>
+          );
+        }}
```

Comment:
I'm a weirdo who likes things small and this is a 100% arbitrary stylistic preference that you're within your right to ignore, but: this can be shrunk considerably

```suggestion
        onCancel={() => setShowVersionModal(false)}
        footer={<Button onClick={() => setShowVersionModal(false)}>Close</Button>}
```

could even give `() => setShowVersionModal(false)` an alias above for simplicity:


```suggestion
        onCancel={closeVersionModal}
        footer={<Button onClick={closeVersionModal}>Close</Button>}
```